app/views/applications/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<h1>Applicant's Name: <%= @application.name%></h1>
+<p> Street Address: <%= @application.street_address%></p>
+<p>City: <%= @application.city%></p>
+<p>State: <%= @application.state%></p>
+<p>Zip Code: <%= @application.zip_code%></p>
+<p>Description: <%= @application.description%></p>
+<p>Status: <%= @application.status%></p>
+
+
+
+<section class="adoptpet">
+
+  <h1>Application Pets</h1>
+    
+  <% @application.pets.each do |pet| %>
+    <h2><%= pet.name%></h2> 
+    
+  <% end %>
+
+</section>
+
+<% @application.pets.each do|pet|%>
+<%= link_to "#{pet.name}", "/pets/#{pet.id}" %>
+<%end%>
+
+
+
+
+<section class="pet">
+  <%= form_with url: "/applications/#{@application.id}", method: :get  do |f| %>
```

Comment:
Did this form ever give you trouble with having `data: {turbo: false}`? It looks like it works on your deployment which is great, but I've seen this cause issues with Turbo before if it's not disabled. Just something to keep in mind! If you see weird issues with your form, add `data: {turbo: false}`

Reply:
we did not get any errors with this form when were working on it but I totally forgot to add the turbo statement so this is a good reminder!